### DISCUSSION TITLE
HTML within WorkflowData

### DISCUSSION THREAD

**Discussion by IanBassi** [microsoft/MIMWAL#99] [Q&A] [answered]
Hi All,

Is it possible to save HTML markup to WorkflowData and get the HTML markup to be used, instead of being displayed as text.  The ParameterTable function returns the HTML, but it gets displayed as below instead of a table when the email is sent out (I added the table tags to the email template).

`<tr><td class="AttributeStyle">DisplayName</td><td class="AttributeModificationStyle">Modify</td><td class="NewValueStyle">Ian Test</td></tr><tr><td class="AttributeStyle">Email</td><td class="AttributeModificationStyle">Modify</td><td class="NewValueStyle">Iantest@contoso.com</td></tr><tr><td class="AttributeStyle">xccountName</td><td class="AttributeModificationStyle">Modify</td><td class="NewValueStyle">IanTest</td></tr>`

My solution to this issue so far has been to use PowerShell to send the email, but I was wondering if there was another way to do this using the send notification?

Thanks,

Ian

**Comment by NileshGhodekar** [comment, marked as answer]
Not that I'm ware of. The markup chars are automatically escaped by MIM Service, so can't think of anything apart from PowerShell workaround that you mentioned.
